src/main/webapp/item-page.html

```diff
@@ -1,52 +1,33 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>Item Page</title>
-    <!-- Bootstrap CSS -->
-    <link
-      rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
-    />
-    <link rel="stylesheet" href="style.css" />
-    <script src="itempagescript.js"></script>
-    <!-- Load an icon library -->
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <!-- Javascript for Bootstrap -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-  </head>
 
-  <body>
-    <div class="container">
-      <img src="media/woww.png" />
-      <h1>The Wolf of Wall Street(2013)</h1>
-      <p><b>R</b></p>
-      <p><b>3 hours</b></p>
-      <p><b>Comedy/Drama</b></p>
-      <br />
-      <p>
-        Description: In 1987, Jordan Belfort (Leonardo DiCaprio) takes an
-        entry-level job at a Wall Street brokerage firm. By the early 1990s,
-        while still in his 20s, Belfort founds his own firm, Stratton Oakmont.
-        Together with his trusted lieutenant (Jonah Hill) and a merry band of
-        brokers, Belfort makes a huge fortune by defrauding wealthy investors
-        out of millions. However, while Belfort and his cronies partake in a
-        hedonistic brew of sex, drugs and thrills, the SEC and the FBI close in
-        on his empire of excess.
-      </p>
-      <footer>
-        <ul id="comment-container"></ul>
-        <form action="/itempagedata" method="POST">
-          <textarea name="text-input">Comment:</textarea>
-          <br /><br />
-          <input type="submit" />
+<head>
+  <meta charset="UTF-8" />
+	<title>Item Page</title>
+	<!-- Bootstrap CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" />
+	<link rel="stylesheet" href="style.css" />
+	<script src="itempagescript.js"></script>
+	<!-- Load an icon library -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+	<!-- Javascript for Bootstrap -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+</head>
+
+<body onload="loadItemPage()">
+	<div class="container">
+	  <div id="item-container"></div>
+		<footer>
+			<ul id="comment-container"></ul>
+			<form>
+			  <textarea name="comment"></textarea>
+				<br/><br/>
+				<button onclick="sendFormData()">Submit</button>
```

Comment:
The indentation here seems to be off, try running prettier to format the html files. The style guide we are using says that regular indentation spaces for HTML and JS files is 2 spaces instead of 4.